### DISCUSSION TITLE
XMLSec 1.3.3-rc1

### DISCUSSION THREAD

**Discussion by lsh123** [lsh123/xmlsec#747] [Announcements]
The [XML Security Library 1.3.3](https://www.aleksey.com/xmlsec-rc/download.html) release includes the following changes:

- (xmlsec-core) Disabled KeyValue and DEREncodedKeyValue XML nodes by default. Use the '--enabled-key-data' option for the xmlsec command line utility or update the 'keyInfoCtx->enabledKeyData' parameter if you need to re-enable these nodes (also see [question 3.5 in the FAQ](https://www.aleksey.com/xmlsec-rc/faq.html#section_3_5)).
- (xmlsec-core) Removed '--enable-size-t' ('size_t' for MSVC builds) option and made 'xmlSecSize' to always be the same as 'size_t'.
- (xmlsec-core) Removed previously deprecated functions, defines, etc.
- (xmlsec-core) Fixed build for libxml2 v2.12.0.
- (xmlsec-openssl) Removed support for OpenSSL 1.1.0 ([end of life in Aug 2016](https://endoflife.date/openssl)). The minimum OpenSSL supported version is 1.1.1; the version 3.0.0 or greater is recommended.
- (xmlsec-nss) Added runtime check for the enabled algorithms in NSS.
- (xmlsec-mscrypto) Removed NT4 support.
- Several other small fixes (see [more details](https://github.com/lsh123/xmlsec/commits/master)).

Please test the release [candidate](https://www.aleksey.com/xmlsec/download/xmlsec1-1.3.3-rc1.tar.gz) ([signature](https://www.aleksey.com/xmlsec/download/xmlsec1-1.3.3-rc1.sig)) and let me know if you see any issues!

<hr /><em>This discussion was created from the release <a href='https://github.com/lsh123/xmlsec/releases/tag/1.3.3-rc1'>XMLSec 1.3.3-rc1</a>.</em>

**Comment by vmiklos** [comment]
Looks good here, integration tests (odfsig on 3 platforms, libreoffice on Linux) found no issues so far.

**Reply by lsh123** [reply]
Thank you!
